src/app/(post-login)/submission/program/[shortName]/registration/page.tsx

```diff
@@ -116,9 +117,21 @@ export default function Register({
     },
   );
 
+  const [uploadClinicalSubmission, mutationStatus] = useMutation(
+    UPLOAD_CLINICAL_SUBMISSION_MUTATION,
+    {
+      onCompleted: (d) => {
+        //setSelectedClinicalEntityType(defaultClinicalEntityType);
```

Comment:
Is this purposefully commented out? Wondering if we need to keep track of functionality that's tied to the API integration / not able to implement right now

Reply:
good catch ty